doc/release_notes/v2.1.0.rst

```diff
@@ -41,6 +41,11 @@ Enhancements
   the image data is converted to signed (:issue:`1149`)
 * Added :attr:`~pydicom.uid.UID.keyword` property for the new UID keywords
   in version 2020d of the DICOM Standard
+* Added testing of the variable names used when setting
+  :class:`~pydicom.dataset.Dataset` attributes and
+  :attr:`~pydicom.config.INVALID_KEYWORD_BEHAVIOR` config option to allow
+  customizing the behaviour when a camel case variable name is used that isn't
```

Comment:
dangit!

Reply:
I can live with it 😄 I couldn't think of anything better.

Reply:
> Dangit! But 'behavior' just feels wrong

"customizing the behaviour" should also feel wrong - should be "customising the behaviour" 😁

Reply:
From [grammarist](https://grammarist.com/spelling/customise-customize/):
> In the U.S. and Canada, it’s customize, customized, customizing, customization, customizable, etc. Outside North America, it’s customise, customised, customising, customisation, customisable, etc.

So, for you it is indeed the correct spelling, while in the U.K. or somewhere down under it's "customise" (as I just learned...) 😄

Reply:
Yeah, Canadian spelling does seem to be a mish-mash of some US, some British-origin.